FEATURES.md

```diff
@@ -0,0 +1,17 @@
+# Proposing Features
+
+A feature request or proposal is an issue opened on a repo in opensearch-project that states an end-user problem, and proposes a solution in the form of a feature. It should meet the following criteria.
+
+- Is a feature.
+- Fits in the scope of the repo and project.
+- Is sufficiently distinct from existing features.
+- Doesn’t overlap or conflict with ongoing work.
```

Comment:
as far as you know.  There's a ton of work happening, so while we want even casual users to propose features, we can't expect them to have the roadmap memorized.

Reply:
I agree. Removing.